app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/models/User.java

```diff
@@ -115,6 +119,37 @@ public void setLastName(String lastname) {
     public Uri getProfilePictureURI(){
         return ppUri;
     }
+    /**
+     * Setter for the profile picture URI
+     * @param uri
+     */
+    public void setProfilePictureURI(Uri uri){
+        ppUri = uri == null ? Uri.EMPTY : uri;
```

Comment:
maybe later on we should use a default picture( like facebook) to be displayed when no user picture and even meal picture is available so it can look nicer instead of Uri.EMPTY

Reply:
Yes, that's a good idea, although i think it would be better to implement that on the backend side, as when a user registers for the frist time, the default profile picture would be put in the user's field.